File: app/src/main/java/abdullah/elamien/bloodbank/ui/SettingsFragment.java
package abdullah.elamien.bloodbank.ui;

import android.os.Bundle;

import androidx.preference.PreferenceFragmentCompat;

import abdullah.elamien.bloodbank.R;


/**
 * Created by AbdullahAtta on 2/16/2019.
 */
public class SettingsFragment extends PreferenceFragmentCompat {
    @Override
    public void onCreatePreferences(Bundle savedInstanceState, String rootKey) {
        addPreferencesFromResource(R.xml.settings);
    }
}

File: app/src/main/java/abdullah/elamien/bloodbank/ui/SettingsFragment.kt
package abdullah.elamien.bloodbank.ui

import abdullah.elamien.bloodbank.R
import android.os.Bundle
import androidx.preference.PreferenceFragmentCompat


/**
 * Created by AbdullahAtta on 2/16/2019.
 */
class SettingsFragment : PreferenceFragmentCompat() {
    override fun onCreatePreferences(savedInstanceState: Bundle?, rootKey: String?) {
        addPreferencesFromResource(R.xml.settings)
    }
}
